Specs/DateTime/English/TimeZoneExtractor.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "Input": "Book me a room at beijing time",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "beijing time",
+        "Type": "timezone"
+      }
+    ]
+  },
+  {
+    "Input": "Book me a room at utc4:30",
+    "NotSupportedByDesign": "python",
```

Comment:
A lot of missing notsupported in JS, no?

Reply:
For now this json is only used in .NET tests so I think it's ok. But for safety, I add the not supported in JS.